src/components/AccountSettingsDashboard/components/ProfileDetailsTab.tsx

```diff
@@ -0,0 +1,61 @@
+import { Trans } from '@lingui/macro'
+import { Button } from 'antd'
+import { Badge } from 'components/Badge'
+import { Callout } from 'components/Callout'
+import { Formik } from 'formik'
+import { User } from 'models/database'
+import { useProfileDetailsTab } from '../hooks/useProfileDetailsTab'
+import { AccountSettingsSchema } from '../lib/AccountSettingsSchema'
+import { AccountSettingsForm } from './AccountSettingsForm'
+
+export const ProfileDetailsTab = ({ user }: { user: User }) => {
+  const { initialValues, lastEmailUpdated, onEmailResendClicked, onSubmit } =
+    useProfileDetailsTab({ user })
+  return (
+    <div className="flex flex-col gap-y-10">
+      <div>
+        <h1 className="text-primary text-2xl dark:text-slate-100">
+          <Trans>Profile details</Trans> <Badge variant="info">Beta</Badge>
+        </h1>
+        <div className="text-secondary">
+          <Trans>
+            Customize your public facing profile and add additional management
+            information.
+          </Trans>
+        </div>
+      </div>
+      <Formik
+        initialValues={initialValues}
+        validationSchema={AccountSettingsSchema}
+        onSubmit={onSubmit}
+      >
+        {({ isSubmitting, dirty }) => (
+          <>
+            <AccountSettingsForm address={user.wallet} />
+            <div className="flex h-9">
+              <Button
+                type="primary"
+                htmlType="submit"
+                form="accountSettings"
+                disabled={!dirty}
+                loading={isSubmitting}
+              >
+                <Trans>Save profile details</Trans>
+              </Button>
+            </div>
+          </>
+        )}
+      </Formik>
+      {lastEmailUpdated && (
+        <Callout.Info>
+          <Trans>
+            Please check {lastEmailUpdated} and verify your new email address.
+            <br />
+            Still no email after a couple of minutes?{' '}
+            <a onClick={onEmailResendClicked}>Click here to resend.</a>
```

Comment:
```suggestion
            <Button type="link" onClick={onEmailResendClicked}>Click here to resend.</Button>
```

Reply:
Is there any benefit to this? I have to manually unset the padding on the link button rather than just use a `a`